override/override.css

```diff
@@ -1,11 +1,13 @@
-.Card {
+.Card, [data-testid="dashcard"] {
   box-shadow: none !important;
 }
 
-[data-testid="embed-frame"] table [data-testid="column-header"], [data-testid="embed-frame"] div:has(> [data-testid="legend-caption"] [data-testid="legend-caption-title"]:empty), [data-testid="embed-frame"] div:has(> [data-testid="legend-caption"] [data-testid="legend-caption-title"]):has(+ div [data-testid="loading-spinner"]) {
+[data-testid="embed-frame"] table [data-testid="column-header"]:only-child, 
```

Comment:
reformated and added `:only-child`, so it apply only we hare displaying a score only / one bollumn